counsel.el

```diff
@@ -4517,6 +4517,71 @@ If there is no such buffer, start a new `shell' with NAME."
                             (reusable-frames . visible)))
     (shell name)))
 
+;;** `counsel-firefox-bookmarks'
+(defvar counsel-firefox-bookmarks-file
+  (car (file-expand-wildcards "~/.mozilla/firefox/*/bookmarks.html"))
+  "Firefox's automatically exported HTML bookmarks file.")
+
+(defface counsel-firefox-bookmarks-tag
+  '((t :inherit font-lock-comment-face))
+  "Face used by `counsel-firefox-bookmarks' for tag."
```

Comment:
Better to write `"...for tags."` than `"...for tag."`

Reply:
Done